File: app/src/main/java/com/sabihamumcu/kelimekutusu/MainActivity.java
package com.sabihamumcu.kelimekutusu;

import android.content.Intent;
import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void yeniKelime(View view) {
        Intent intent = new Intent(MainActivity.this, KelimeEkle.class);
        startActivity(intent);
    }

    public void testEt(View view) {
        Intent intent = new Intent(MainActivity.this, TestEt.class);
        startActivity(intent);
    }

    public void listele(View view) {
        Intent intent = new Intent(MainActivity.this, Listele.class);
        startActivity(intent);
    }

    public void Gonder(View view) {
        Intent intent = new Intent(MainActivity.this, Send.class);
        startActivity(intent);
    }

}

File: app/src/main/java/com/sabihamumcu/kelimekutusu/MainActivity.kt
package com.sabihamumcu.kelimekutusu

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.sabihamumcu.kelimekutusu.MainActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun addWord(view: View?) {
        val intent = Intent(this@MainActivity, AddWord::class.java)
        startActivity(intent)
    }

    fun testing(view: View?) {
        val intent = Intent(this@MainActivity, Testing::class.java)
        startActivity(intent)
    }

    fun list(view: View?) {
        val intent = Intent(this@MainActivity, ListResult::class.java)
        startActivity(intent)
    }

    fun send(view: View?) {
        val intent = Intent(this@MainActivity, Send::class.java)
        startActivity(intent)
    }
}